arcgis-ios-sdk-samples/Utility Network/Perform valve isolation trace/PerformValveIsolationTraceViewController.swift

```diff
@@ -0,0 +1,261 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class PerformValveIsolationTraceViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The button to start the trace.
+    @IBOutlet weak var traceButton: UIBarButtonItem!
+    /// The button to choose a utility category for filter barriers.
+    @IBOutlet weak var categoryButton: UIBarButtonItem!
+    /// The switch to control whether to include isolated features in the trace results when used in conjunction with an isolation trace.
+    @IBOutlet weak var isolationSwitch: UISwitch!
+    /// The 3-line label to display navigation status.
```

Comment:
```suggestion
    /// The label to display trace status.
```
🤦 Uhh forget to change the copied comment.